notebooks/tel_and_site/subsys_req_ver/mthexapod/SITCOM-2190_apply_fft_to_motor_current.ipynb

```diff
@@ -0,0 +1,743 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #28.        signal = detrend(signal)
</pre><p>I suggest to check the signal before and after detrending just to make sure that you are not loosing something (I don't think so, but better to check) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-sitcom/notebooks_vandv/pull/195/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>